test/testpartner.ts

```diff
@@ -1,59 +1,119 @@
+import { asMap, asObject, asOptional, asString, asUnknown } from 'cleaners'
 import {
   addEdgeCorePlugins,
+  EdgeFakeUser,
   EdgeSwapRequest,
   lockEdgeCorePlugins,
   makeFakeEdgeWorld
 } from 'edge-core-js'
+import fs from 'fs'
 
 import edgeExchangePlugins from '../src'
+import { avaxCurrencyInfo } from './fakeAvaxInfo'
 import { btcCurrencyInfo } from './fakeBtcInfo'
 import { makeFakePlugin } from './fakeCurrencyPlugin'
 import { ethCurrencyInfo } from './fakeEthInfo'
 
+const DUMP_USER = false
+const DUMP_USER_FILE = './test/fakeUserDump.json'
+
+const asDateStr = (raw: string): Date => new Date(raw)
```

Comment:
This is not a cleaner. Cleaners only take `any`, not `string`. I think you want `asDate`, which also handles the string conversion like you have here.

Reply:
done